File: src/main/java/ch/junamin/models/Cell.java
package ch.junamin.models;

public class Cell {
    int row, col;
    char value;

    public Cell(int row, int col, char value) {
        this.row = row;
        this.col = col;
        this.value = value;
    }

    @Override
    public String toString() {
        return value + "(" + row + "," + col + ")";
    }
}

File: dotnet-core/BimaruApi/domain/Cell.cs
namespace BimaruApi.Domain
{
    public class Cell
    {
        public int Row { get; }
        public int Col { get; }
        public char Value { get; }

        public Cell(int row, int col, char value)
        {
            Row = row;
            Col = col;
            Value = value;
        }

        public override string ToString()
        {
            return $"{Value}({Row},{Col})";
        }
    }
}
